forced_photometry/multiband_photometry.ipynb

```diff
@@ -132,44 +92,20 @@
     "\n",
     "### codes that may or may not already be installed in the platform\n",
     "try:\n",
```

Comment:
This is an empty `try/except` that prevents the cell from executing. Looks like it can be deleted.

Reply:
thanks for spotting it, I was messing a lot with manual line commits in text mode and didn't spot this one. 

I may also rework the tractor imports, so to make most of the notebook runnable locally without a tractor install (which would e.g. help me a lot in hunting down the bug Marziye identified with the non-centralized cutouts)

Reply:
I've ended not pushing the missing tractor try/excepts, but just keep them around locally. So this should be ready to go now.